packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -599,6 +599,8 @@ class BuildIOSArchiveCommand extends _BuildIOSSubCommand {
             return 'release-testing';
           case 'development':
             return 'debugging';
+          case 'enterprise':
```

Comment:
Instead of throwing can the default fall through to returning the `method`, since I think the args are already enforcing it's in the list? That was we doing have to update this to add new non deprecated methods.

Reply:
oh yes good point